### PR TITLE
Add an instruction to build plugin

### PR DESCRIPTION
**Pull request by tiratatp**
I saw that the steps to build plugin is not straightforward. Is it possible to add these to the README file?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,3 +76,15 @@ datasources:
       accessKey: '<your access key>'
       secretKey: '<your secret key>'
 ```
+
+## How to build this datasource
+You will need to install [Node.js](https://nodejs.org/en/), [Yarn](https://yarnpkg.com/), [Go](https://golang.org/), and [Mage](https://magefile.org/) first.
+1. `yarn install --frozen-lockfile`
+1. `rm -rf node_modules/@grafana/data/node_modules`
+1. `npm run build`
```
Comment (ryantxu) [inline README.md:84]:
```suggestion
1. `yarn dev`
```
This will be the most forgiving way to build/run the environment

Comment (tiratatp) [reply README.md]: Fixing in my latest commit. Please review them.

File: README.md [hunk 2]
```diff
@@ -76,3 +76,15 @@ datasources:
...
       secretKey: '<your secret key>'
 ```
+
+## How to build this datasource
+You will need to install [Node.js](https://nodejs.org/en/), [Yarn](https://yarnpkg.com/), [Go](https://golang.org/), and [Mage](https://magefile.org/) first.
+1. `yarn install --frozen-lockfile`
+1. `yarn dev` — will build the frontend changes
+1. `mage build:backend` — will build the backend changes
+1. (Optional) `mage -v buildAll` — this is optional if you need backend plugins for other platforms
+1. [Sign the plugin](https://grafana.com/docs/grafana/latest/developers/plugins/sign-a-plugin/), or configure Grafana to [load the unsigned plugin](https://grafana.com/docs/grafana/latest/plugins/plugin-signatures/#allow-unsigned-plugins).
```
Comment (ryantxu) [inline README.md:86]: This one I'm not sure about because 3rd parties can not actually sign...  Perhaps we just say to run grafana in developer mode?

**Review by ryantxu — APPROVED** [review]

**Comment by tiratatp** [comment]
Hey team, can we merge this?

File: README.md (hunk 2 above)
Comment (tiratatp) [reply README.md:86]: Can they sign a plugin for a particular hostname?

Comment (ryantxu) [reply README.md:86]: not really -- the signature is the thing that says it is official.  If it is for development, just run grafana in dev mode and all is well.  

If they want their own version of the plugin, they need to change the id and then can sign it with a personal key (with hostname etc), but we don't let 3rd parties sign plugins they do not "own"

**Review by ryantxu — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (tiratatp) [reply README.md:86]: Let me fix this in another PR then.

**Pull request merged**